### PR TITLE
tests added

### PR DESCRIPTION
**Pull request by codecoder005**

### CODE REVIEW

File: src/test/java/com/orgofarmsgroup/util/AppConstants.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package com.orgofarmsgroup.util;
+
+public final class AppConstants {
+    public static class API_URI {
+        public static class USERS_API_URI {
+            public static final String USERS_ROOT = "/users";
+        }
+    }
```
Comment (Raakhi130937) [inline src/test/java/com/orgofarmsgroup/util/AppConstants.java:8]: Can we follow better naming conventions for classes here?

Comment (Raakhi130937) [inline src/test/java/com/orgofarmsgroup/util/AppConstants.java:8]: Can we follow better naming conventions for classes here?

File: src/test/java/com/orgofarmsgroup/controller/UserControllerTest.java [hunk 2]
```diff
@@ -24,7 +25,7 @@ class UserControllerTest {
     @DisplayName("users root api")
     void testUsers() throws Exception {
         mockMvc.perform(
-                get("/users")
+                get(AppConstants.API_URI.USERS_API_URI.USERS_ROOT)
         ).andDo(print()).andExpect(status().isOk());
```
Comment (Raakhi130937) [inline src/test/java/com/orgofarmsgroup/controller/UserControllerTest.java:29]: Can you verify the data received in the API response as well?

**Review by Raakhi130937 — COMMENTED** [review]
Please look into the below comments.

File: src/test/java/com/orgofarmsgroup/util/AppConstants.java (hunk 1 above)
Comment (codecoder005) [reply src/test/java/com/orgofarmsgroup/util/AppConstants.java:8]: sure.

File: src/test/java/com/orgofarmsgroup/controller/UserControllerTest.java (hunk 2 above)
Comment (codecoder005) [reply src/test/java/com/orgofarmsgroup/controller/UserControllerTest.java:29]: sure

**Review by Raakhi130937 — APPROVED** [review]
looks fine. approved.

**Pull request merged**